openedx/core/djangoapps/models/course_details.py

```diff
@@ -43,6 +47,10 @@ def __init__(self, org, course_id, run):
         self.enrollment_start = None
         self.enrollment_end = None
         self.syllabus = None  # a pdf file asset
+        self.title = ""
```

Comment:
Since these new fields are in the `ABOUT_ATTRIBUTES` list, they get persisted down below on line 249 of this file via the `update_about_item` method.